src/content/en/updates/_deprecation-blurb.md

```diff
@@ -0,0 +1,6 @@
+{# wf_md_include #}
+
+<aside>
```

Comment:
I'd change this to `<aside class="note">` - that'll give it some slightly nicer styling.